src/components/Inventory/InventoryTable.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import strings from 'src/strings';
-import { Table, TableColumnType } from '@terraware/web-components';
+import { TableColumnType } from '@terraware/web-components';
 import { Box, Grid, useTheme } from '@mui/material';
 import { SearchResponseElement } from 'src/api/search';
 import InventoryCellRenderer from './InventoryCellRenderer';
 import { InventoryFiltersType } from './InventoryFiltersPopover';
 import PageSnackbar from 'src/components/PageSnackbar';
 import { APP_PATHS } from 'src/constants';
 import Search from './Search';
+import Table from 'src/components/common/table';
```

Comment:
Why are this all import changes (from web-components to common) ?

Reply:
Because the versions in `common` now pass extra parameters to the components. The other option here would have been to add the parameters individually to every `<Table>` in the application, but it seemed cleaner to put them in one place.

https://github.com/terraware/terraware-web/pull/1068/files#diff-387ed938b72221d7f368435bd9c9e197815d14e9774e37bf7829892820c51def may make this clearer.

Reply:
Oh, excellent! Thank you. Looks good!